src/helpers/misc/Eth.re

```diff
@@ -0,0 +1,42 @@
+type t = BN.bn;
```

Comment:
This File isn't needed in this PR, but I am busy integrating this into the rest of the codebase, so since we are iterating quite fast I am leaving it in. (will delete all the code that this is meant to replace soon)